game/scripts/npc/items/farming/item_travel_origin.txt

```diff
@@ -84,11 +84,6 @@
         "var_type"                                        "FIELD_FLOAT"
         "assist_stickiness"                               "0.1"
       }
-      "05"
-      {
-        "var_type"                                        "FIELD_INTEGER"
-        "creep_bounty_percent"                            "30"
```

Comment:
Will removing these values make them behave as if they're 100 or 0? Have you tested it?

Reply:
nvm didn't know fools gold was already removed from the sparks